Source/EasyNetQ.Management.Client/ManagementClient.cs

```diff
@@ -31,7 +31,7 @@ public class ManagementClient : IManagementClient
         private readonly HttpClient httpClient;
 
         private readonly Regex urlRegex =
-            new Regex(@"^(http|https):\/\/.+\w$", RegexOptions.Compiled | RegexOptions.Singleline);
+            new Regex(@"^(http|https):\/\/\[*.+\w\]*$", RegexOptions.Compiled | RegexOptions.Singleline);
```

Comment:
`\[*` accept more than one `[`? probably better `\[?`

Reply:
Yes that's a very good point. Updated.